api/src/main/java/org/openmrs/module/htmltojson/htmltojson/HtmlFormUtil.java

```diff
@@ -145,6 +147,16 @@ public static ObjectNode getFormSchemaJson(String formUuid, ResourceFactory reso
 						} else {
 							
 							ObjectNode dataPointJson = generateJsonObjectForHtmlDataPoint(element);
+							if (currentIndex >= 3) {
```

Comment:
What is the basis of this? Why 3?

Reply:
Yeah it seems like a magic number but when I looked at the `elementTags` and the HFE file itself I noticed that this would be the best index to start of and since I always need to go to indexes back to tentatively get the label am also preventing with this an `indexOutOfBounds` exception.

Reply:
We can add a comment to help those reviewing the code now and even later.